infobar-with-cta/template.ts

```diff
@@ -12,16 +12,13 @@ export class InfobarWithCTATemplate implements CampaignTemplateComponent {
     ])
     style: StyleField = { label: "Light on Dark", className: "evg-light-on-dark" };
 
+    @subtitle("Optional text field")
     messageText: string = "Infobar Message Text";
 
-    messageVisibility: boolean = true;
-
     @title('CTA Text')
```

Comment:
Could you make this minor update too?
```suggestion
    @title("CTA Text")
```